test/unit/utils/states_test.js

```diff
@@ -55,13 +55,13 @@ describe('states', function () {
     })
 
     it('should get display name for connector state', function () {
-      expect(states.getDisplayNameForConnectorState({status: 'submitted'})).to.equal('In progress')
-      expect(states.getDisplayNameForConnectorState({status: 'created'})).to.equal('In progress')
-      expect(states.getDisplayNameForConnectorState({status: 'success'})).to.equal('Success')
-      expect(states.getDisplayNameForConnectorState({status: 'success'}, 'refund')).to.equal('Refund success')
-      expect(states.getDisplayNameForConnectorState({status: 'error'}, 'refund')).to.equal('Refund error')
-      expect(states.getDisplayNameForConnectorState({status: 'timedout'})).to.equal('Timed out')
-      expect(states.getDisplayNameForConnectorState({status: 'declined'})).to.equal('Declined')
+      expect(states.getDisplayNameForConnectorState({ status: 'submitted' })).to.equal('In progress')
+      expect(states.getDisplayNameForConnectorState({ status: 'created' })).to.equal('In progress')
+      expect(states.getDisplayNameForConnectorState({ status: 'success' })).to.equal('Success')
+      expect(states.getDisplayNameForConnectorState({ status: 'success' }, 'refund')).to.equal('Refund success')
+      expect(states.getDisplayNameForConnectorState({ status: 'error' }, 'refund')).to.equal('Refund error')
+      expect(states.getDisplayNameForConnectorState({ status: 'timedout' })).to.equal('Timed out')
+      expect(states.getDisplayNameForConnectorState({ status: 'declined' })).to.equal('Declined')
```

Comment:
is it worth adding in 'capturable' to all the tests in this class?

Reply:
yes it is. added. thanks.